integration-tests/oidc-wiremock/pom.xml

```diff
@@ -18,6 +18,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-oidc</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Any chance we can avoid adding this dependency ? You can add a custom mock in the test itself

Reply:
Yeah, that was mistake, it was unnecessary.